apps/ejabberd/src/ejabberd_sm.erl

```diff
@@ -608,10 +608,10 @@ do_route(From, To, Packet) ->
       Reason :: any().
 do_route_no_resource_presence_prv(From, To, Packet, Type, Reason) ->
     is_privacy_allow(From, To, Packet) andalso ejabberd_hooks:run_fold(
-                                                 roster_in_subscription,
-                                                 To#jid.lserver,
-                                                 false,
-                                                 [To#jid.user, To#jid.server, From, Type, Reason]).
+        roster_in_subscription,
```

Comment:
I have that feeling the indentation here is not correct. Maybe splitting line 610 into two will help .